services/hanke-service/build.gradle.kts

```diff
@@ -64,6 +64,8 @@ plugins {
 	kotlin("plugin.jpa") version "1.6.21"
 	idea
 	id("com.github.ben-manes.versions") version "0.42.0"
+	id("jacoco")
+	id("org.sonarqube") version "4.2.1.3168"
```

Comment:
Do we need the Sonarqube plugin, since we're not implementing the SonarCloud upload?

Reply:
Based on the azure devops pipeline, it is not needed. Removed.